server/database/sqlc.yaml

```diff
@@ -24,6 +24,18 @@ overrides:
         go_type:
           import: github.com/speakeasy-api/gram/server/internal/urn
           type: Tool
+      - column: http_tool_definitions.tool_urn
```

Comment:
You need these to be pointers unless you’ve already backfilled the tables, otherwise rows with null urns from the database will error on unmarshalling.